_layouts/default.html

```diff
@@ -51,12 +57,9 @@ <h1 class="noselect">South Eugene Robotics Team</h1>
             </li>
             {% endif %}
             {% endfor %}
-            <li style="height: 36px">
-                <form action="https://www.paypal.com/cgi-bin/webscr" method="post" target="_blank" style="margin-top: 5px">
-                    <input type="hidden" name="cmd" value="_s-xclick">
-                    <input type="hidden" name="hosted_button_id" value="XAPA9VJHWAE94">
```

Comment:
You can't get rid of these two lines, they're how PayPal knows which donation page to redirect to.

Reply:
Fixed 👍